### DISCUSSION TITLE
AdonisJS webpack-encore DELETED my assets files

### DISCUSSION THREAD

**Discussion by adityanurdin** [adonisjs/core#3106] [Help]
![image](https://user-images.githubusercontent.com/37808974/131246632-80ff9d7c-55fc-4f09-b3e7-dd0015e38f18.png)

why my assets files always deleted by webpack-encore while compiling (npm run dev) like on the pic, while creating the Adonisjs project I choose "true" for configure webpack-encore. I newbie at webpack or something like that, please help me I'm so stuck here. thanks for the answer and I appreciate it. 😄😄

**Comment by McSneaky** [comment]
It deletes all old assets, otherwise with every new build you would end up adding new resources, but never deleting old ones.

Don't put stuff manually to `/public/assets`. Use `/resources/` folder instead of `/public/`
